compendium/DeclarativeServices/test/gtest/TestFixture.hpp

```diff
@@ -314,7 +317,30 @@ namespace test
         cppmicroservices::Framework framework;
         cppmicroservices::BundleContext context;
     };
-
+    class tGenericDSAndCASuite : public tGenericDSSuite
+    {
+      public:
+        void SetUp() override
+        {
+            tGenericDSSuite::SetUp();
+            ::test::InstallAndStartConfigAdmin(context);
+            auto svcRef = context.GetServiceReference<cppmicroservices::service::cm::ConfigurationAdmin>();
+            ASSERT_TRUE(svcRef);
+            configAdmin = context.GetService(svcRef);
+            ASSERT_TRUE(configAdmin);
+        }
+        void
+        TearDown() override
+        {
+            if (configAdmin)
+            {
+                configAdmin.reset();
+            };
```

Comment:
Is is necessary to call `reset()`? destroying the `tGenericDSAndCASuite` object does the same thing.

Reply:
Will fix